cfgrammar/src/lib/header.rs

```diff
@@ -18,19 +18,18 @@ pub enum HeaderErrorKind {
     ExpectedToken(char),
     DuplicateEntry,
 }
+// Indicates a value prefixed by an optional namespace.
+// `Foo::Bar` with optional `Foo` specified, becomes (Some(("Foo", ...)), ("Bar", ...))
+// Alternately just `Bar` alone without a namespace is represented by (None, ("Bar", ...)).
+pub type Namespaced = (Option<(String, Span)>, (String, Span));
```

Comment:
I wonder if this has got fiddly enough that it should be a `struct`? I must admit that I tend to confuse pairs when the LHS and RHS types are more-or-less the same.

Reply:
I'll give it a shot